scripts/drone/events/cron.star

```diff
@@ -77,3 +78,25 @@ def slack_job_failed_step(channel, image):
             'status': 'failure'
         }
     }
+
+def post_to_grafana_com_step():
+    return {
+            'name': 'post-to-grafana-com',
+            'image': publish_image,
+            'environment': {
+                'GRAFANA_COM_API_KEY': from_secret('grafana_api_key'),
+                'GCP_KEY': from_secret('gcp_key'),
+            },
+            'depends_on': ['grabpl'],
+            'commands': ['./bin/grabpl publish grafana-com --edition oss'],
+        }
+
+def post_to_grafana_com_pipeline():
+    return cron_job_pipeline(
+            cronName='grafana-com-nightly',
+            name='grafana-com-nightly',
+            steps=[
+                download_grabpl_step(),
+                post_to_grafana_com_step(),
+            ])
+
```

Comment:
why a pipeline in its own right, not a step on the end of an existing one?

Reply:
Yeah right, because that was an old local branch and didn't include @fridgepoet and my refactoring yet 😅 
Will change. Good catch 👍